src/simulator/perftest.py

```diff
@@ -165,6 +165,11 @@ def run_single_test(self, test):
         for key, value in test.items():
             if not key == 'test':
                 coordinator_params[key] = value
+        # Add member user to worker parameters
+        hosts = load_hosts(inventory_path=inventory_path, host_pattern="all:!mc:!load_balancers")
+        if hosts:
+            user = ssh_user(hosts[0])
+            coordinator_params['members_user'] = user
```

Comment:
I'm a bit lost on this without context - could you explain what this is doing please?

Reply:
It is just adding the user param from the iventory.yaml -- like ec2-user or ubuntu. We require it for setting the owner of the dir